servers/quarkus-common/src/main/java/org/projectnessie/quarkus/config/QuarkusNessieAuthorizationConfig.java

```diff
@@ -13,7 +13,7 @@
  * See the License for the specific language governing permissions and
  * limitations under the License.
  */
-package org.projectnessie.server.config;
+package org.projectnessie.quarkus.config;
```

Comment:
This config also appears to be used only inside `quarkus-server`... so why move it?